src/wp-includes/class-wp-theme-json.php

```diff
@@ -764,6 +764,16 @@ private function get_block_classes( $style_nodes ) {
 				}
 			}
 
+			// Reset default browser margin on the root body element.
+			// We set this on the root selector **before** generating the ruleset
+			// from the `theme.json`. This is to ensure that if the `theme.json` declares
+			// `margin` in its `spacing` declaration for the `body` element then these
+			// user-generated values take precedence in the CSS cascade.
+			// See: https://github.com/WordPress/gutenberg/issues/36147.
```

Comment:
Per Core's coding standards, multiline comments should be in this format:

```suggestion
			/*
			 * Reset default browser margin on the root body element.
			 * We set this on the root selector **before** generating the ruleset
			 * from the `theme.json`. This is to ensure that if the `theme.json` declares
			 * `margin` in its `spacing` declaration for the `body` element then these
			 * user-generated values take precedence in the CSS cascade.
			 * @link https://github.com/WordPress/gutenberg/issues/36147.
			 */
```